packages/fscomponents/src/components/SingleLineForm.tsx

```diff
@@ -1,19 +1,19 @@
 import React, { PureComponent } from 'react';
-import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-
+import {StyleProp, StyleSheet, Text, TouchableOpacity, View, ViewStyle} from 'react-native';
 import { Form, FormLabelPosition } from './Form';
+import {Dictionary} from '@brandingbrand/fsfoundation';
 
 export interface SingleLineFormProps {
-  fieldsTypes: any;
-  fieldsOptions?: any;
-  fieldsStyleConfig?: any;
+  fieldsTypes: Dictionary;
+  fieldsOptions?: Dictionary;
+  fieldsStyleConfig?: Dictionary;
   labelPosition?: FormLabelPosition;
-  onSubmit?: (value: any) => void;
-  submitButtonStyle?: any;
-  submitTextStyle?: any;
-  submitText?: any;
-  value?: any;
-  style?: any;
+  onSubmit?: <T>(value: T) => void;
+  submitButtonStyle?: StyleProp<ViewStyle>;
+  submitTextStyle?: StyleProp<ViewStyle>;
+  submitText?: () => void;
+  value?: string;
```

Comment:
We should check but I think this is going to be an object since the list of fields is an object.